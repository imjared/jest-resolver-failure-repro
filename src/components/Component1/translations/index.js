import { i18n } from 'utils/i18n';

export default i18n({
  'en-US': require(`./en-US.json`),
});
