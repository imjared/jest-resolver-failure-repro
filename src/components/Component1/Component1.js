import PropTypes from 'prop-types';
import React from 'react';

import translations from './translations';

class Component1 extends React.Component {
  render() {
    return (
      <div>
        {translations[`en-US`].content}
      </div>
    );
  }
}

export default Component1;
