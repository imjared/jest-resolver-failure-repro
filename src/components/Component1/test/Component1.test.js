import React from 'react';
import { render } from 'enzyme';

import Component1 from 'components/Component1/Component1';

describe(`Component 1`, () => {
  it(`renders`, () => {
    const component = render(<Component1 />);
    expect(component).toMatchSnapshot();
  });
});
