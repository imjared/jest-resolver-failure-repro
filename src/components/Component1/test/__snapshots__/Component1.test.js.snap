// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component 1 renders 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum dolorum, impedit architecto similique laboriosam accusamus.",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "div",
        "next": null,
        "parent": null,
        "prev": null,
        "root": [Circular],
        "type": "tag",
      },
    ],
    "name": "root",
    "next": null,
    "parent": null,
    "prev": null,
    "type": "root",
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xmlMode": false,
  },
}
`;
